tests/web/test_block_tendermint.py

```diff
@@ -73,29 +71,3 @@ def test_get_blocks_by_txid_endpoint_returns_empty_list_not_found(client):
     res = client.get(BLOCKS_ENDPOINT + '?transaction_id=123')
     assert res.status_code == 200
     assert len(res.json) == 0
-
-
-@pytest.mark.bdb
-def test_get_blocks_by_txid_endpoint_returns_400_bad_query_params(client):
-    res = client.get(BLOCKS_ENDPOINT)
-    assert res.status_code == 400
-
-    res = client.get(BLOCKS_ENDPOINT + '?ts_id=123')
-    assert res.status_code == 400
-    assert res.json == {
-        'message': {
-            'transaction_id': 'Missing required parameter in the JSON body or the post body or the query string'
-        }
-    }
-
-    res = client.get(BLOCKS_ENDPOINT + '?transaction_id=123&foo=123')
-    assert res.status_code == 400
-    assert res.json == {
-        'message': 'Unknown arguments: foo'
-    }
-
-    res = client.get(BLOCKS_ENDPOINT + '?transaction_id=123&status=123')
-    assert res.status_code == 400
-    assert res.json == {
-        'message': 'Unknown arguments: status'
```

Comment:
Why is the error response message different between these two tests? The one in `test_blocks` asserts 'Unknown arguments: status'. How did these ever both pass?